di_website/common/mixins.py

```diff
@@ -134,3 +134,13 @@ class SectionBodyMixin(models.Model):
 
     class Meta:
         abstract = True
+
+
+class CallToActionMixin(models.Model):
+    call_to_action_title = models.CharField(max_length=255, null=True, blank=True, default="Receive a pdf version", help_text="Leave title field blank to remove this call to action")
+    call_to_action_body = models.TextField(null=True, blank=True, default="We shall email you a pdf version")
+    call_to_action_button_text = models.CharField(max_length=255, null=True, blank=True, default="Signup to receive a copy in your inbox")
+    call_to_action_button_url = models.URLField(max_length=255, null=True, blank=True, default="https://us11.list-manage.com/subscribe?u=a829237ca0cf1470615c7f059&id=ce30e2af0f")
```

Comment:
Remove the defaults, and for lines this long, put the properties on their own lines ...

Reply:
Updated